src/main/apiClient.ts

```diff
@@ -914,7 +917,8 @@ export default class ApiClient {
           ...authParams,
           ...challenge
         },
-        withCookies: true
+        withCookies: params.method === 'session',
+        accessToken: (params.method === 'access_token') ? params.accessToken : undefined
```

Comment:
This changes current behaviour, should we allow not to provide a `method` param and default to `sesison` in this case?

Reply:
I think we can just set `withCookies` to false if any access token is passed and do without `method`.